pkg/util/util_test.go

```diff
@@ -119,6 +119,11 @@ func (s *getAddrIPSuite) TestShouldRetIPV4(c *C) {
 		IP: net.ParseIP("192.168.1.2"),
 	}
 	c.Assert(getAddrDefaultIP(&addr), Equals, "192.168.1.2")
+
+	addr2 := net.IPAddr{
```

Comment:
what's the propose of this added test, seems same as the "192.168.1.2" before

Reply:
one is `net.IPNet` while new added is `net.IPAddr`